src/main/java/org/test/dto/UserResponseDto.java

```diff
@@ -0,0 +1,30 @@
+package org.test.dto;
+
+public class UserResponseDto {
+    private String name;
```

Comment:
let's add id too

Reply:
it's not a problem if we ill have a separate fields firstName and lastName instead of splitting concatenate strings during mapping